### PR TITLE
docs(website): tables from source rules to Biome rules

### PR DESCRIPTION
**Pull request by ematipico**
## Summary

This PR creates a new page containing three tables:
- a table from from ESLint rules to Biome rules
- a table from `typescript-eslint` rules to Biome rules
- a table from ESLint plugin rules to Biome rules

I called this page "Linter sources", but I admit that the title isn't that great, so I would like some suggestion about naming 😄 

## Test Plan

Check the end result of the webpage

### CODE REVIEW

**Comment by Conaclos** [comment]
> I called this page "Linter sources", but I admit that the title isn't that great, so I would like some suggestion about naming

ESlint rules? This makes clear that the page is about ESlint correspondance.

What do you think about creating a section for each source instead of grouping plugins in a same table?

File: website/src/content/docs/linter/linter-sources.mdx [hunk 1]
```diff
@@ -0,0 +1,163 @@
+---
+title: Linter sources
+description: A page that maps lint rules from other sources to Biome
+---
+
```
Comment (Conaclos) [inline website/src/content/docs/linter/linter-sources.mdx:5]: We could add a warning saying that most of the rules doesn't provide the equivalent options provided by ESlint?

**Comment by ematipico** [comment]
> ESlint rules? This makes clear that the page is about ESlint correspondence.

That isn't true, because we have clippy rules too. Check the updated tables

**Comment by ematipico** [comment]
I updated the table with your suggestions. I am not really sure about the footnotes, they add some bloat.

**Comment by Conaclos** [comment]
We could rename `lint rules` to `rules` (this page is already under the `linter` navigation title, it is clear from the contexts that this refers to lint rules), and `linter sources` to `rule sources`.

**Comment by ematipico** [comment]
> Also, we should use a single footnote for inspired rule.

It doesn't work, I already tried. We can't have multiple carets (numbers next to the word) that point to the same footnote. It's probably how the footnotes in markdown work.

**Comment by ematipico** [comment]
Check my latest update

**Review by Conaclos — APPROVED** [review]

**Pull request merged**